### ISSUE TITLE
Contacts sync and import won't work

### ISSUE THREAD

**Issue by BDub-Hay** [pierre-alain-b/rainloop-nextcloud#13]
Hello, I am trying to synchronize my Nextcloud contacts in the Nextcloud Rainloop app but it isn't working.  I get an "unknown" error when I hit the synchronize button.   I get a "Import error (invalid file format)" when i try to upload a .vcf file created from my Nextcloud contacts app.  I have been searching my apache logs but can't find any errors that would help me out.  If anybody could lend a hand here I would greatly appreciate it.  Thanks.

Brian.

**Comment by pierre-alain-b** [comment]
Do you see something if you activate Rainloop logging?

**Comment by BDub-Hay** [comment]
How would I do that.  I went to the admin page in Nextcloud and the admin panel in Rainloop and I don't see how to activate logging.  

Brian

**Comment by pierre-alain-b** [comment]
See here:
https://github.com/pierre-alain-b/rainloop-nextcloud#how-to-activate-rainloop-logging-and-then-find-logs

**Comment by BDub-Hay** [comment]
I see this message pop up in the log:

`[17:58:19.490][1d876c79] SQL[DATA]: exception 'PDOException' with message 'SQLSTATE[HY000] [1049] Unknown database 'rainloop'' in /path/to-owncloud/owncloud/apps/rainloop/app/rainloop/v/1.11.0.203/app/libraries/RainLoop/Common/PdoAbstract.php:104`

I logged into my database and did a "show databases" and confirmed there isn't a rainloop database.  Is that the root of my problem?

**Comment by pierre-alain-b** [comment]
Could very well be. The contacts are stored in the database. Are you sure the database user used for Rainloop has the right to open the database?

**Comment by BDub-Hay** [comment]
problem solved.  I created a database that the php script is looking for.  

A couple of things I noticed in my search for answers:
   1. Under the "contacts" section in the .ini file i had to modify to enable logging, I had to switch the root user to the database admin user i created (I don't like using the root account to do much of anything without me physically logging in and entering comands)
   2.  I also had to enter the admin database user password under the same "contacts" section.

Thank you for the help, I really do appreciate it.

Hopefully this helps somebody else looking for the same help.

Brian
